.github/workflows/deploy-customer.yml

```
@@ -81,29 +48,57 @@ jobs:
+                echo "ERROR: JAR file not found!"
+                ls -la /home/ec2-user/
+                exit 1
+              fi
+              
+              echo "Found JAR file: $JAR_FILE"
+              
+              (
+                export AWS_REGION=ap-northeast-2
+                export AWS_ACCESS_KEY=${{ secrets.AWS_ACCESS_KEY_ID }}
+                export AWS_SECRET_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)